dcap/types/src/tcb.rs

```diff
@@ -132,6 +135,26 @@ impl TryFrom<&Certificate> for TcbInfo {
     }
 }
 
+impl<T: AsRef<[u8]>> TryFrom<&Quote3<T>> for TcbInfo {
```

Comment:
There was a TODO at https://github.com/mobilecoinfoundation/attestation/blob/345dd82f8ece2feda66fc5847d29f52866e48a9e/verifier/src/evidence.rs#L125

And as I was working on the Sim Quote in the [MobileCoin](https://github.com/mobilecoinfoundation/mobilecoin) repo I needed similar functionality so figured it's time to migrate here.